src/main/java/sirius/biz/storage/layer3/downlink/ssh/BridgeFileSystem.java

```diff
@@ -82,7 +83,11 @@ public Path getPath(String first, String... more) {
         if (Strings.isEmpty(first)) {
             return new BridgePath(virtualFileSystem.root(), this);
         }
-
+        if (more.length == 0) {
+            VirtualFile topLevelDirectory = virtualFileSystem.root().resolve(first);
+            topLevelDirectory.assertExists();
+            return new BridgePath(topLevelDirectory, this);
+        }
         // We only seem to have to handle the root access - we throw the following exception if our
```

Comment:
No clue why we abort here, instead of doing something like `virtualFileSystem.root().resolve(first + "/" + String.join("/", more))`

Reply:
I did not wan't to do a major change shorly before the release. So this sticks closer to existing implementation